Bearded.UI/Controls/CompositeControl.cs

```diff
@@ -17,6 +17,8 @@ public CompositeControl()
             Children = children.AsReadOnly();
         }
 
+        public static CompositeControl ClickThrough() => new CompositeControl {IsClickThrough = true};
```

Comment:
Given the implementation, do we need this at all? If so, can we rename it to CreateClickThrough() or NewClickThrough()?

Reply:
I see the point for not needing this, but I think creating a "transparent" composite control to wrap existing controls is a pretty common use case.